app/views/partner_requests/_item.html.erb

```diff
@@ -1,5 +1,5 @@
 <tr>
-  <td><%= form.select :name, POSSIBLE_ITEMS.slice(*@valid_items).invert.collect, {include_blank: 'Select an item'}, {class: 'form-control'} %></td>
+<td><%= form.select :name, POSSIBLE_ITEMS.slice(*@valid_items).invert.sort_by{|k,v|k}, {include_blank: 'Select an item'}, {class: 'form-control'} %></td>
```

Comment:
Nice :100:

Reply:
Looks great! Did you boot the app with these changes and make sure it still looks right?

Reply:
Yes.  It still looks good!